int32 id
int32 positiveLaneNum
int32 negativeLaneNum
bool twoWay
int32[] positiveLaneIds
int32[] negativeLaneIds
int32 mode
float32 n_as
float32 n_kph
float32 p_as
float32 p_kph
float32 length
ItriPoint[] points
ConnectedNavgRoad[] heads
ConnectedNavgRoad[] tails
